Documentation/supermuc.rst

```diff
@@ -34,24 +40,52 @@ where ddddd is an arbitrary 5-digital port number.
     
 With ddddd the same port number as before.
 
-4. Create SSH key by typing (use a non-empty passphrase, not too long as you will need to type it often)
+3. Create SSH key by typing (use a non-empty passphrase, not too long as you will need to type it often)
 
 ::
 
   ssh-keygen -t rsa 
 
-5. Go to https://github.com/settings/ssh, add a new SSH key, pasting the public key you just created on supermuc  ~/.ssh/id_rsa.pub. 
-Logout of supermuc and log back in (ssh supermucNG). You should now be able to clone SeisSol including the submodules using:
+4. Go to https://github.com/settings/ssh, add a new SSH key, paste the public SSH key you just created on supermuc  ~/.ssh/id_rsa.pub. For example,
+
+Title
 
+::
+
+  supermucNG 
+
+Key
+
+::
+
+  ssh-rsa AAB3Nzy2......zjZXKTq 
+
+5. Logout of supermuc and log back in with command
+
+::
+
+  ssh supermucNG 
+  
+You should now be able to clone SeisSol including the submodules using:
 
 ::
 
   git clone git@github.com:SeisSol/SeisSol.git
   cd SeisSol
   git submodule update --init
 
-Pay attention to the git clone address ('https://github.com/' replaced by 'git@github.com:'). 
-If it works, you will see several lines of ‘cloning ….’.
+Pay attention to the typical git clone address ('https://github.com/' is replaced now by 'git@github.com:'). 
```

Comment:
Pay attention to the change in the git address ('https://github.com/' is now replaced by 'git@github.com:').